src/scripts/scripts.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module "@gimenete/type-writer";
+declare module "table-builder";
```

Comment:
//TODO: Create proper modules for third party dependencies

Reply:
Not an issue anymore after this agreement: https://github.com/octokit/webhooks.js/pull/113#discussion_r416975529